packages/dataviews/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Introduce new `array` type in DataForm fields. [#71136](https://github.com/WordPress/gutenberg/pull/71136)
```

Comment:
This is a small nit, but it's important to me that we differentiate between a field type and a DataForm Edit control:

```suggestion
- Introduce a new `array` DataForm Edit control that supports multi-selection. [#71136](https://github.com/WordPress/gutenberg/pull/71136)
```